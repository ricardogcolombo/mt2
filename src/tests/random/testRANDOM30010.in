300 10
1 212 4 165 105
1 135 120 179 27
1 197 52 22 84
1 194 117 30 108
1 63 38 51 122
1 214 12 53 98
1 48 0 116 101
1 17 110 151 29
1 66 72 136 40
1 55 77 233 128
