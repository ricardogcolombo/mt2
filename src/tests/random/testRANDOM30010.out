-nan
-nan
-nan
-nan
-nan
-nan
-nan
0.000000
-nan
-nan
-nan
-nan
-nan
-nan
1.000000
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
1.000000
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
0.000000
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
0.000000
-nan
1.000000
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
0.000000
-nan
-nan
0.000000
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
0.000000
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
0.000000
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
0.000000
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
1.000000
-nan
-nan
-nan
-nan
-nan
-nan
-nan
0.000000
-nan
1.000000
-nan
-nan
-nan
1.000000
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
1.000000
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
1.000000
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
1.000000
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
0.000000
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
-nan
